langchain/vectorstores/elastic_vector_search.py

```diff
@@ -498,6 +500,8 @@ def knn_hybrid_search(
         fields: Optional[
             Union[List[Mapping[str, Any]], Tuple[Mapping[str, Any], ...], None]
         ] = None,
+        vector_query_field: Optional[str] = 'vector',
```

Comment:
should both of these just be stored on the object?

Reply:
Actually I'm with you now. I'll move them to the init and update this

Reply:
@dev2049 I moved the two field names as optional args toin `__init__` 
Let me know if that was what you had in mind